docs/getting-started/security-ui.asciidoc

```diff
@@ -91,6 +91,14 @@ The Alerts page allows you to view and manage all alerts to monitor activity wit
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
+[float]
+=== Findings page
+
+The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
+
+[role="screenshot"]
+image::cloud-native-security/images/findings-page.png[The Findings page]
```

Comment:
Image still needs a little TLC. If it's not possible to re-do with higher res (maybe dev provided it), no worries. Just add the border so it matches the other screenshots on the UI page.